pallets/domains/src/types.rs

```diff
@@ -10,7 +10,10 @@ pub(crate) type DomainName<T> = BoundedVec<u8, <T as Config>::MaxDomainLength>;
 pub(crate) type InnerValueOf<T> = InnerValue<<T as frame_system::pallet::Config>::AccountId>;
 pub(crate) type OuterValue<T> = BoundedVec<u8, <T as Config>::MaxOuterValueLength>;
 
-pub(crate) type BoundedDomainsVec<T> = BoundedVec<DomainName<T>, <T as Config>::DomainsInsertLimit>;
+pub(crate) type RecordKey<T> = BoundedVec<u8, <T as Config>::MaxRecordKeySize>;
+pub(crate) type RecordValue<T> = BoundedVec<u8, <T as Config>::MaxRecordKeySize>;
+
+pub(crate) type BoundedDomainsVec<T> = BoundedVec<DomainName<T>, <T as Config>::MaxRecordValueSize>;
```

Comment:
Is it a typo? Should it be as it was previously?
```suggestion
pub(crate) type BoundedDomainsVec<T> = BoundedVec<DomainName<T>, <T as Config>::DomainsInsertLimit>;
```